scripts/zones/Wajaom_Woodlands/IDs.lua

```diff
@@ -35,6 +35,17 @@ zones[xi.zone.WAJAOM_WOODLANDS] =
     },
     mob =
     {
+        CHIGOES                =
+        {
+            ['Marid'] =
+            {
+                16986118,
```

Comment:
For this zone, we can use this instead of hard-coded IDs:
```
-- ------------------------------------------------------------
-- Wajaom Woodlands (Zone 51)
-- ------------------------------------------------------------

-- Fished
INSERT INTO `mob_spawn_points` VALUES (16986113,'Red_Smoocher','Red Smoocher',1,1,1,1,0);
INSERT INTO `mob_spawn_points` VALUES (16986114,'Azoth_Apsaras','Azoth Apsaras',2,1,1,1,0);
INSERT INTO `mob_spawn_points` VALUES (16986115,'Red_Osculator','Red Osculator',3,1,1,1,0);
INSERT INTO `mob_spawn_points` VALUES (16986116,'Kissing_Leech','Kissing Leech',4,1,1,1,0);
INSERT INTO `mob_spawn_points` VALUES (16986117,'Mercurial_Makara','Mercurial Makara',5,1,1,1,0);

INSERT INTO `mob_spawn_points` VALUES (16986118,'Chigoe','Chigoe',6,291.141,-24.079,115.188,173);
INSERT INTO `mob_spawn_points` VALUES (16986119,'Chigoe','Chigoe',6,176.445,-27.911,88.478,95);
INSERT INTO `mob_spawn_points` VALUES (16986120,'Chigoe','Chigoe',6,196.772,-24.035,100.669,2);
INSERT INTO `mob_spawn_points` VALUES (16986121,'Chigoe','Chigoe',6,139.364,-26.447,298.630,48);
INSERT INTO `mob_spawn_points` VALUES (16986122,'Chigoe','Chigoe',6,139.809,-26.213,301.022,94);
INSERT INTO `mob_spawn_points` VALUES (16986123,'Lesser_Colibri','Lesser Colibri',7,663,-18,198,127);
INSERT INTO `mob_spawn_points` VALUES (16986124,'Lesser_Colibri','Lesser Colibri',7,676,-18,181,127);
```
```lua
['Marid'] = GetTableOfIDs("Chigoe", 5)
```